Index: src/main/java/dev/sheengo/onlinemedclinic/configs/ThreadSafeCollections.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package dev.sheengo.onlinemedclinic.configs;\n\nimport java.util.ArrayList;\nimport java.util.Collections;\nimport java.util.List;\n\npublic interface ThreadSafeCollections {\n    List<Integer> id = Collections.synchronizedList(new ArrayList<>());\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/dev/sheengo/onlinemedclinic/configs/ThreadSafeCollections.java b/src/main/java/dev/sheengo/onlinemedclinic/configs/ThreadSafeCollections.java
--- a/src/main/java/dev/sheengo/onlinemedclinic/configs/ThreadSafeCollections.java	
+++ b/src/main/java/dev/sheengo/onlinemedclinic/configs/ThreadSafeCollections.java	
@@ -4,6 +4,6 @@
 import java.util.Collections;
 import java.util.List;
 
-public interface ThreadSafeCollections {
+public class ThreadSafeCollections {
     List<Integer> id = Collections.synchronizedList(new ArrayList<>());
 }
